sopel/cli/run.py

```diff
@@ -344,11 +344,9 @@ def get_configuration(options):
     try:
         settings = utils.load_settings(options)
     except config.ConfigurationNotFound as error:
-        print(
-            "Welcome to Sopel!\n"
-            "I can't seem to find the configuration file, "
-            "so let's generate it!\n")
-        settings = utils.wizard(error.filename)
+        print("Welcome to Sopel!")
+        error.hint = "To start the configuration wizard, run `sopel configure`"
```

Comment:
Regardless of how this message gets into the user's terminal, I can imagine that it would be useful to put the full command line, e.g. copy whatever `-c` arg was passed here and replicate it in the example. That might also be too much to do at once, and you're well within your rights to tell me "do that later".

Reply:
It seemed Annoying since argparse can't go backwards, but I guess it's probably okay to only bother with `-c` and `--config-dir`... Will do.

Reply:
Actually... Only the parsed `options` object gets passed to `get_configuration()`, so I don't think I can get the defaults, so I don't think I can cleanly omit options the user didn't specify... `if "-c" not in sys.argv`? Or just let the user figure it out?

Reply:
Hmm... You're right, that is Annoying. The hell with it, at least for now; just changing one word will be enough:

```suggestion
        error.hint = "To start the configuration wizard, use `sopel configure`"
```

While "run" implies "this is what you should type", "use" implies "this is the right tool" only.